src/moin/cli/_tests/test_scrapy_crawl.py

```diff
@@ -61,7 +61,6 @@ def test_home_page(self, crawl_results):
         expected = CrawlResultMatch(url='/Home')
         assert expected.match(r), f'unexpected redirect for / {r}'
 
-    @pytest.mark.skip
```

Comment:
I didn't see any logs from this failure, so enabled without modification, please provide logs from failures if this test continues to be troublesome